File: Dopamine.Common/Controls/SubMenuButton.cs
using System.Windows;
using System.Windows.Controls;

namespace Dopamine.Common.Controls
{
    public class SubMenuButton : RadioButton
    {
        static SubMenuButton()
        {
            DefaultStyleKeyProperty.OverrideMetadata(typeof(SubMenuButton), new FrameworkPropertyMetadata(typeof(SubMenuButton)));
        }
    }
}

File: Dopamine.Controls/SubMenuButton.vb
Public Class SubMenuButton
    Inherits RadioButton

#Region "Construction"
    Shared Sub New()
        'This OverrideMetadata call tells the system that this element wants to provide a style that is different than its base class.
        'This style is defined in themes\generic.xaml
        DefaultStyleKeyProperty.OverrideMetadata(GetType(SubMenuButton), New FrameworkPropertyMetadata(GetType(SubMenuButton)))
    End Sub
#End Region
End Class